### PR TITLE
Fixes #6770: Rework makefile to make it more standard

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/6770

### CODE REVIEW

**Comment by Kegeruneku** [comment]
[Magic](http://i.imgur.com/YsbKHg1.gif)

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -57,37 +57,11 @@ URL: http://www.rudder-project.org
 
 Group: Applications/System
 
-Source1: rudder-agent.init
```
Comment (Kegeruneku) [inline rudder-agent/SPECS/rudder-agent.spec]: Hey we still need those to create a SRPM that works ! :)

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

**Comment by Kegeruneku** [comment]
Still missing a slibclean

**Comment by peckpeck** [comment]
PR updated

**Comment by Kegeruneku** [comment]
Good :) let's try this !

**Pull request merged**